ui/js/controllers/perf/graphs.js

```diff
@@ -35,9 +35,8 @@ perf.controller('GraphsCtrl', [
     '$window', '$q', '$timeout',
     function GraphsCtrl($state, $stateParams, $scope, $rootScope,
         $uibModal, $window, $q, $timeout) {
-        var availableColors = ['maroon', 'navy', 'pink', 'turquoise', 'brown',
-            'red', 'green', 'blue', 'orange', 'purple'];
-
+        var availableColors = ['darkseagreen', 'lightseagreen', 'darkslateblue', 'darkgreen', 'steelblue', 'darkorchid', 'blue', 'darkcyan'];
```

Comment:
could this be a ``const`` instead of a ``var``?

Reply:
yup, I'll change it.